Added doc for Angler AI (Actions) (#6890)
segmentio/segment-docs: src/connections/destinations/catalog/actions-angler-ai/index.md

> ---
> title: Angler AI (Actions) Destination
> ---

Reviewer: For context about this recommendation: 
- `beta`: adds the destination to lists of destinations in beta, adds a beta badge in the catalog, etc.
- `id`: the internal ID Segment uses to refer to this destination. Adding it here lets us autogenerate documentation from the catalog submission (Available Actions, for example.)

```suggestion
---
title: Angler AI (Actions) Destination
beta: true
id: 668d1cb2a1dcc5ad33228d92
---
```

{% include content/plan-grid.md name="actions" %}

> [Angler AI](https://getangler.ai/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners){:target="_blank”} helps direct to consumer brands radically improve prospecting and customer lifetime value using the power of AI. Gain a full view of your customer base, deploy custom audiences with precision and measure the true ROI of your campaigns.

Reviewer: ```suggestion
[Angler AI](https://getangler.ai/?utm_source=segmentio&utm_medium=docs&utm_campaign=partners){:target="_blank”} helps direct-to-consumer brands radically improve prospecting and customer lifetime value using the power of AI. Gain a full view of your customer base, deploy custom audiences with precision, and measure the true ROI of your campaigns.
```

> This destination is maintained by <Angler AI>. For any issues with the destination, [contact their Support team](mailto:developers@getangler.ai).

Reviewer: ```suggestion
This destination is maintained by Angler AI. For any issues with the destination, [contact the Angler AI Support team](mailto:developers@getangler.ai).
```

## Getting started

1. From your workspace's [Destination catalog page](https://app.segment.com/goto-my-workspace/destinations/catalog){:target="_blank”} search for "Angler AI".
2. Select Angler AI and click **Add Destination**.
3. Select an existing Source to connect to Angler AI (Actions).

> 4. Go to the [Angler AI dashboard](https://getangler.ai){:target="_blank"}, find and copy the **Workspace ID** and **Access Token**. This iniformation can alaso be provided by the Angler AI account manager.

Reviewer: ```suggestion
4. Go to the [Angler AI dashboard](https://getangler.ai){:target="_blank"}, find and copy the **Workspace ID** and **Access Token**. This information can also be provided by your Angler AI account manager.
```

> 5. Enter the **Workspace ID** and **Access Token** in the Angler AI destination settings in Segment.

Reviewer: ```suggestion
5. Return to the Segment app and enter the **Workspace ID** and **Access Token** in your Angler AI's destination settings page.
```

{% include components/actions-fields.html %}

### Event Mappings

> A default list of event mappings will be applied when the destination is added to your workspace. The mapping may need to modified depending on the event schema in your Segment workspace. Please refer the [Angler AI Documentation](https://docs.getangler.ai/docs/using-gtm-to-setup-events?utm_source=segmentio&utm_medium=docs&utm_campaign=partners){:target="_blank”} for details about the events that can be sent to Angler AI.

Reviewer: ```suggestion
A default list of event mappings are applied when you add the destination to your workspace. You might need to modify the default mappings, depending on the event schema in your Segment workspace. Please refer the [Angler AI Documentation](https://docs.getangler.ai/docs/using-gtm-to-setup-events?utm_source=segmentio&utm_medium=docs&utm_campaign=partners){:target="_blank”} for details about the events that you can send to Angler AI.
```
